frontend/src/app/commonComponents/Checkboxes.tsx

```diff
@@ -89,7 +89,7 @@ const Checkboxes = (props: Props) => {
   return (
     <div
       className={classnames(
-        "usa-form-group padding-0",
```

Comment:
I remember adding this because without it the COVID-19 form symptoms list styling was off... Sorry about that!
<img width="708" alt="Screenshot 2024-10-29 at 07 27 35" src="https://github.com/user-attachments/assets/c12cc0aa-c5fc-49d0-8d48-5234a7fb2b42">


I looked into it further and if you replace [this line](https://github.com/CDCgov/prime-simplereport/blob/12574a3b6c0c0a86f4b6a3ad78460d67406d07a8/frontend/src/app/testQueue/TestCardForm/diseaseSpecificComponents/CovidAoEForm.tsx#L108) in the CovidAoEForm with 

```
className={`grid-row grid-gap ${showSymptomSelectionError ? "margin-left-0" : ""}`}
```
it should fix this problem in the covid form! 😅